app/views/shared/_sign_options_nav.html.erb

```diff
@@ -0,0 +1,17 @@
+<ul class = 'sign-nav-ul'>
+  <% if user_signed_in? %>
+    <li class = "sign-nav-list">
+      <%= link_to "Sign Out", destroy_user_session_path, method: :delete %>
+    </li>
+    <li class = "sign-nav-list">
+      <%= link_to "Edit Profile", edit_user_registration_path %>
+    </li>
+  <% else %>
+    <li class = "sign-nav-list">
+      <%= link_to "Sign In", new_user_session_path %>
+    </li>
+    <li class = "sign-nav-list">
+      <%= link_to "Sign Up", new_user_registration_path %>
+    </li>
+  <% end %>
+</ul>
```

Comment:
- [ ] Optional suggestions

- We don't need to add any logic inside the .erb file, we use the helper folders to create methods with logic and called inside the .erb file. You can refer to this comment to modify the files that have logic inside the .erb file.

Read [this](https://www.toptal.com/ruby-on-rails/top-10-mistakes-that-rails-programmers-make) for more information

Reply:
@Emmyn5600  

Thank you a lot!
Your suggestions helps me a lot and learn from them! 🙏